hscontrol/auth.go

```diff
@@ -456,12 +456,12 @@ func (h *Headscale) handleNewNode(
 		resp.AuthURL = fmt.Sprintf(
 			"%s/oidc/register/%s",
 			strings.TrimSuffix(h.cfg.ServerURL, "/"),
-			registerRequest.NodeKey,
```

Comment:
I think we should use NodeKey - or we might break fast user switching

Reply:
We need a test for fast user switching.

Node keys can change.